framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceScenesHelper.java

```diff
@@ -86,6 +89,29 @@
      */
     private final Set<String> modifiedScenes = new HashSet<>();
 
+    /**
+     * A map from the description of an ATypeElement to the corresponding TypeMirror.
+     *
+     * <p>AScene doesn't carry around the basetypes of annotated types, but this is needed to output
+     * stub files, such as when outputting the parameters of a method. {@link
+     * #updateTypeElementFromATM} intercepts updates to ATypeElements that add annotations, and
```

Comment:
To "intercept" is to prevent something from happening.  That is not what `updateTypeElementsFromATM` does.